frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -284,4 +255,42 @@ function BookInfo:onShowBookCover(file)
     end
 end
 
+function BookInfo:getCurrentPageLineWordCounts()
+    if self.ui.rolling then
+        local Screen = require("device").screen
```

Comment:
Oh, this one is nearly everywhere at top of files, and nearly never in a function.

Reply:
You said it: it does check every time if the module has been loaded.
So, having it at the top avoid that checks every time your function is called. (even if that check time is peanuts/small, it must be just checking if a thing is in a table, and not worth fighting).

But this is not why I insist on these: it's just for consistency, whether it's good or bad, in all our modules, we have many standard require= at top, it's not worth changeing that and start inlining them now as you will edit modules.
(Again, except for heavy or rarely used modules, which are best require()d when they are about to be used.)

Reply:
It makes it simple to glance at the top of the file to see what it uses. It's a fairly common pattern for that reason.